eth2/types/src/beacon_state.rs

```diff
@@ -108,12 +124,12 @@ where
     pub start_shard: u64,
     pub randao_mixes: FixedVector<Hash256, T::EpochsPerHistoricalVector>,
     #[compare_fields(as_slice)]
-    active_index_roots: FixedVector<Hash256, T::EpochsPerHistoricalVector>,
+    pub active_index_roots: FixedVector<Hash256, T::EpochsPerHistoricalVector>,
```

Comment:
Needed this `pub` hack for `PartialBeaconState`, and I'm inclined to leave it in because AIR and compact committees are going away soon anyway.

Reply:
Aaah, but I did it for `slashings` too... not sure.